.ci/install_kata.sh

```diff
@@ -21,15 +21,15 @@ echo "rust image is default for Kata 2.0"
 "${cidir}/install_kata_image.sh" "${tag}"
 
 echo "Install Kata Containers Kernel"
-"${cidir}/install_kata_kernel.sh" "${tag}"
+"${cidir}/install_kata_kernel.sh" -t "${tag}"
```

Comment:
In `.ci/install_kata.sh`, `tag` is empty by default. As a result, if not defined, it will have this:
```
$ .ci/install_kata_kernel.sh -t 
.ci/install_kata_kernel.sh: option requires an argument -- t
INFO: Installing 'vanilla' kernel
(....)
```
It will install the vanilla kernel (this is what we want) but it prints the error about missing argument to `-t`. So my suggestion is to check if `tag` is empty then pass `-t vanilla` (or not pass `-t` at all).

Reply:
Apart from that, I didn't know what `tag` was meant for so I did some archeology work to figure out the commit where it was introduced:
```
commit 9d868f8306a1059e103d88b22dc5d1bd35c7db75
Author: Jianyong Wu <jianyong.wu@arm.com>
Date:   Wed Aug 7 02:30:52 2019 -0400

    tests: install kata to specific version
    
    Some new features maybe not supported in some hosts
    with old kernel, so let older version of kata installed
    easier by adding a version argument.
    
    Fixes: #1908
    Signed-off-by: Jianyong Wu <jianyong.wu@arm.com>
```
The original author wanted `tag` to mean a kata version, passed as a parameter to `install_kata.sh`. Here the meaning is changed to accommodate the installation of non-default kernel and qemu types. IMHO this seems wrong.

Could we discuss a better way to instruct the scripts to install assets for "vanilla", experimental and CC use cases?

Cc @devimc @jcvenegas you may be interested on this... :)

Reply:
I submitted a new PR: https://github.com/kata-containers/tests/pull/4668 and `tag` will use `vanilla` as default value.